.github/workflows/code-coverage.yml

```diff
@@ -4,7 +4,18 @@ jobs:
   comment-coverage-difference:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Changed files
+        id: changed
+        run: |
+          echo "::set-output name=all::$(git diff --name-only ${{ github.event.pull_request.base.ref }}...${{ github.event.pull_request.head.ref }} | xargs)"
+      - name: Get paths
+        id: relevant-paths
+        run: |
+          echo "::set-output name=frontend-paths::$(echo ${{ steps.changed.outputs.all }} | grep -Eo 'public\/app\/plugins\/datasource\/(grafana-azure-monitor-datasource|cloud-monitoring|cloudwatch)' | sort -u )"
```

Comment:
this (the path) needs to be an input, if not, this will stop working on external plugins (as far as I can see)

Reply:
Yes, the path would be something like `src\/` in plugins